.github/workflows/upload_release_asset.yml

```diff
@@ -0,0 +1,33 @@
+name: Upload Release Asset
+
+on:
+  workflow_dispatch:
```

Comment:
Okay, I read about it. This looks great! 
So from the repository you can start the release process by clicking on the actions.

But do you have to provide the URL? It cannot detect the releases of repository?

Reply:
Actually this file is for the Release Robot, so you can't start the process by hand, but you can run the Release Robot on your local machine and it will trigger this action for you and will provide a URL. The reason for this decision - we don't want to give credentials of confluence and Jira, for example,  to GitHub. So the credentials are stored locally and this action is only responsible for the GitHub release currently

Reply:
I see, yes makes sense.

But when you say credentials, it is only the Github token, similar to Sonar? It should be possible to encode it in Github secrets.